doc/source/_static/custom.css

```diff
@@ -0,0 +1,34 @@
+/* colors */
+
+:root {
+    --pst-color-primary: 19, 156, 90;
+    --pst-color-active-navigation: 19, 156, 90;
+    --pst-color-h2: var(--color-text-base);
+}
+
+/* buttons */
+
+.button>p>a {

```

Comment:
Is this actually used somewhere at the moment?

Reply:
Nope, lazy copy&paste :).